File: Src/java/cqf-fhir/src/main/java/org/cqframework/fhir/utilities/exception/IGInitializationException.java
package org.cqframework.fhir.utilities.exception;

/**
 * This exception is thrown whenever there is an issue initializing the tooling from a source IG.
 */
public class IGInitializationException extends RuntimeException {
    static final long serialVersionUID = 1L;

    public IGInitializationException() {
        super();
    }

    public IGInitializationException(String message) {
        super(message);
    }

    public IGInitializationException(String message, Throwable cause) {
        super(message, cause);
    }

    public IGInitializationException(Throwable cause) {
        super(cause);
    }
}

File: Src/java/cqf-fhir/src/main/kotlin/org/cqframework/fhir/utilities/exception/IGInitializationException.kt
package org.cqframework.fhir.utilities.exception

/**
 * This exception is thrown whenever there is an issue initializing the tooling from a source IG.
 */
class IGInitializationException : RuntimeException {
    constructor(message: String?, cause: Throwable?) : super(message, cause)

    companion object {
        const val serialVersionUID: Long = 1L
    }
}
